docs/advanced-topics/message-system/backward-compatibility.md

```diff
@@ -0,0 +1,10 @@
+---
+id: backward-compatibility
+title: Backward Compatibility
```

Comment:
🔥   This is weird... Backward Compatibility should be in the release notes. May need to consider moving this entirely to release notes and add a note to the upgrade/migration guides? Also it should not be a stand alone page, but turned into a section on the main message system intro page if you want to keep this info with it. It doesn't mention what it is backward compatibly info for...messaging system? Specific message system APIs?

Reply:
this section came direct form the RFC for Messaging , it's a section on backward compatibility of ServerRpc and ClientRpc. Your right though it doesn't belong here at the moment so should be deleted.

Reply:
removed

Reply:
Section removed